diff --git a/Makefile b/Makefile
index 802a6b9..68e4658 100644
--- a/Makefile
+++ b/Makefile
@@ -52,18 +52,6 @@ install: lib
 uninstall:
 	ocamlfind remove asp
 
-precheck:
-	@echo -n "checking OPAM compiler switch ... "
-	@test $$(opam switch  show) = "4.11.1+BER"  \
-      || test $$(opam switch  show) = "4.07.1+BER"  \
-      || test $$(opam switch  show) = "4.04.0+BER"  \
-      || (echo 1>&2 "Please use OPAM switch 4.04.0+BER, 4.07.1+BER or 4.11.1+BER"; exit 1)
-	@echo "ok"
-	@echo -n "checking packages ... "
-	ocamlfind query letrec >/dev/null \
-             || (echo >&2 "\n\n\tpackage letrec is not installed\n\n\tCheck the README for instructions\n"; exit 1)
-	@echo "ok"
-
 clean:
 	$(OCAMLBUILD) -clean
 
