diff --git a/Makefile b/Makefile
index e2bd35b..0af6ee2 100644
--- a/Makefile
+++ b/Makefile
@@ -36,11 +36,4 @@ clean:
 
 tests.byte: letrec.cma lib_test/tests.ml
 
-check-compiler:
-	@test $$(opam show ocaml-variants --field=installed-version) = "4.14.1+BER"  \
-      || test $$(opam show ocaml-variants --field=installed-version) = "4.11.1+BER"  \
-      || test $$(opam show ocaml-variants --field=installed-version) = "4.07.1+BER"  \
-      || test $$(opam show ocaml-variants --field=installed-version) = "4.04.0+BER"  \
-      || (echo 1>&2 "Please use OPAM switch 4.04.0+BER, 4.07.1+BER, 4.11.1+BER or 4.14.1+BER"; exit 1)
-
 .PHONY: check-compiler all clean test
 